### PR TITLE
Rory/visual changes article card

### PR DESCRIPTION
**Pull request by rorrrr**
### List of Changes

- Design overhaul of Home component
- Added Light/Dark mode toggle and corresponding themes
- Added script for mock-dev environment when not connected to API
- Updated ReadMe

### Next steps

- Check `npm run dev` still works. Couldn't check this without API.
- Add preferred theme to local storage
- Add light/dark theme throught (only Home currently)
 

### Test Coverage 
Screenshot 2020-10-19 at 17 08 08
Not a single failing test.

### CODE REVIEW

File: components/ArticleCard.jsx [hunk 1]
```diff
@@ -17,27 +17,35 @@ export const ArticleCard = ({ article, index }) => {
...
-          className={styles.imgHolder}
-          style={{
-            backgroundImage: `url(${article.img})`,
-          }}
-        />
+          <div
+            className={styles.imgHolder}
+            style={{
+              backgroundImage: `url(${article.img})`,
+            }}
```
Comment (rorrrr) [inline components/ArticleCard.jsx:27]: Screenshot 2020-10-19 at 12 04 25

wide mode solo number tweak

File: pages/api/getLatestNews.js [hunk 2]
```diff
@@ -1,13 +1,27 @@
 const { pool } = require("../../config/db");
 const { subDays, format } = require("date-fns");
+const mockLatestNews = require("./fixtures/mockGetLatestNews.json");
```
Comment (rorrrr) [inline pages/api/getLatestNews.js:3]: mocks the api call if running npm run dev-mock

File: components/ArticleCard.jsx (hunk 1 above)
Comment (rorrrr) [reply components/ArticleCard.jsx:27]: Screenshot 2020-10-19 at 12 07 09

under 900px view including mobile has number still in article title.

File: components/Home.module.css [hunk 3]
```diff
@@ -1,5 +1,10 @@
-.homeWrapper {
-  height: calc(100vh - 60px);
+.lightWrapper {
+  text-align: center;
+  background-color: white;
+  color: #20364c;
```
Comment (rorrrr) [inline components/Home.module.css:4]: removed height from homeWrapper. homeWrapper was overflowing, but looked normal before
Screenshot 2020-10-19 at 16 56 29
 as the bodybackground=homebackground

File: pages/index.jsx [hunk 4]
```diff
@@ -15,6 +15,6 @@ export default function Index() {
         />
       </Head>
       <Home />
-    </Layout>
+    </LayoutProvider>
   );
```
Comment (rorrrr) [inline pages/index.jsx:19]: Feel free to rename the providers/components. Was winging it here.

**Review by parkingpete — APPROVED** [review]
lgtm. I've changed a bit the getLatest bit as after your changes the usual http request was not sorting articles correctly and changed few contrasts on ArticleCard. Looks absolutely amazing, thanks a lot! next step, I need to persist the theme choice for returning users ;)

**Pull request merged**